feature-libs/cart/added-to-cart-toast/components/added-to-cart-toast/added-to-cart-toast.component.ts

```diff
@@ -0,0 +1,185 @@
+import {
+  Component,
+  HostBinding,
+  OnDestroy,
+  OnInit,
+  Renderer2,
+} from '@angular/core';
+import {
+  CartAddEntrySuccessEvent,
+  EventService,
+  ProductService,
+  RoutingService,
+  WindowRef,
+} from '@spartacus/core';
+import { combineLatest, Observable, of, Subscription } from 'rxjs';
+import { filter, map, switchMap } from 'rxjs/operators';
+import { AddedToCartToastConfig } from '../../added-to-cart-toast-config';
+import { CartToastItem, CART_TOAST_STATE } from '../../core/models';
+import { AddedToCartToastService } from '../../core/services';
+
+@Component({
+  selector: 'cx-added-to-cart-toast',
+  templateUrl: './added-to-cart-toast.component.html',
+})
+export class AddedToCartToastComponent implements OnInit, OnDestroy {
+  protected subscription = new Subscription();
+
+  quantity: number;
+
+  customClass?: string;
```

Comment:
I am not sure but in components, we are not using `?` in variables

Reply:
I was basing this component off of the `PopoverComponent` and it has a `?` there. I could remove it if it's better